docs/swagger.json

```diff
@@ -0,0 +1,651 @@
+{
+	"swagger": "2.0",
+	"info": {},
+	"basePath": "/corellia/api",
+	"tags": [
+		{
+			"name": "Corellia"
+		}
+	],
+	"paths": {
+		"/version": {
+			"get": {
+				"tags": [
+					"Corellia"
+				],
+				"summary": "returns version",
+				"description": "",
+				"operationId": "getVersion",
+				"produces": [
+					"text/plain"
+				],
+				"parameters": [],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		},
+		"/contracts": {
+			"post": {
+				"tags": [
+					"Corellia"
+				],
+				"summary": "Saves a contract",
+				"description": "",
+				"operationId": "saveContract",
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"in": "body",
+						"name": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/Contract"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		},
+		"/documents": {
+			"post": {
+				"tags": [
+					"Corellia"
+				],
+				"summary": "Saves a document",
+				"description": "",
+				"operationId": "saveDocument",
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"in": "body",
+						"name": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/Document"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"Address": {
+			"type": "object",
+			"required": [
+				"city",
+				"houseNumber",
+				"street",
+				"zipCode"
+			],
+			"properties": {
+				"street": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"houseNumber": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"zipCode": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"city": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				}
+			}
+		},
+		"Agent": {
+			"type": "object",
+			"required": [
+				"agentNumber",
+				"email"
+			],
+			"properties": {
+				"agentNumber": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"email": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				}
+			}
+		},
+		"Company": {
+			"type": "object",
+			"required": [
+				"address",
+				"legalForm",
+				"name",
+				"uidNumber"
+			],
+			"properties": {
+				"name": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
```

Comment:
We have been told max 70

Reply:
There are going to be a lot of changes. Look at this as a first try, maybe a base for discussions. We needed a contract for our firewall/security team to open up a gate for you. Hopefully next week we can provide a better version with documentation tags, correct field lenghts and optimized inheritance. Certainly we will include your feedback, thank you very much!

Reply:
Cool. Thanks.   I think the most pressing question for us is the one about this.

Can we clarify here the order of operations, as it's unclear.

In previous discussions with Frank it was.

Upload document, which returns a token
Make policy request, including token for document so you know what to copy in background.
Here, it looks like

Upload policy?
Upload documents with policy Id reference.
Could you clarify? I think at least we should be saying in policy what document as expected

As we want to start looking at the integration on our end

Reply:
should be answered with aboves comment